### ISSUE TITLE
ESlint errors when using fullpage.js extensions

### ISSUE THREAD

**Issue by alvarotrigo** [alvarotrigo/react-fullpage#123]
Some people reported errors when using `create-react-app`.

**Comment by alvarotrigo** [comment]
For some reason it doesn't seem to be possible to ignore the file by using the file `.eslintignore` as pointed out [here](https://stackoverflow.com/a/48714377/1081396) and [here](https://github.com/facebook/create-react-app/issues/2339#issuecomment-303403944).

So the solution is to add the comments `/* eslint-disable */` and `/* eslint-enable */` wrapping the extension file.

For example:

```js
/* eslint-disable */
/*!
 * fullpage.js Scroll Horizontally Extension 0.0.4.1 for fullPage.js v3
 * https://github.com/alvarotrigo/fullPage.js
 *
 * @license This code has been bought from www.alvarotrigo.com/fullPage/extensions/ and it is not free to use or distribute.
 * Copyright (C) 2016 alvarotrigo.com - A project by Alvaro Trigo
 */
window.fp_scrollHorizontallyExtension=[EXTENSION CODE HERE]
/* eslint-enable */

**Comment by alvarotrigo** [comment]
This issue has been fixed in `Scroll Horizontally Extension 0.0.4 `
